hydromt_wflow/wflow.py

```diff
@@ -1859,24 +1858,39 @@ def write_forcing(
                         self.write_config()  # re-write config
                         fn_out = fn_default_path
 
+            # Check if all dates between (starttime, endtime) are in all da forcing
+            start = pd.to_datetime(self.get_config("starttime"))
+            end = pd.to_datetime(self.get_config("endtime"))
+            missings = False
+            for da in self.forcing.values():
+                if "time" in da.coords:
```

Comment:
time should always be in forcing right? Why did you add this if statement?

Reply:
Apparently spatial_ref ends up as a da in self.forcing and it does not have time dimension